src/app/Main/Body/QueryPage/Services/TableServices/calculateRelativeTableData.service.ts

```diff
@@ -11,17 +11,11 @@ function calculateRelative(part: number, total: number): string {
 export function calculateRelativeTableData(
     objList: Record<string, string>[], 
     nrOfSelectedIsolates: number,
-    tableHeadRelative: string,
-    isRow: boolean,
 ): Record<string, string>[] {
     const statisticTableDataRel: Record<string, string>[] = [];
     objList.forEach((obj) => {
         const relativeRow: Record<string, string> = { name: obj.name };
-        if (isRow) {
-            relativeRow.name = obj.name

```

Comment:
Just for interest: Is obj.name === undefined or something if isRow === false?

Reply:
obj.name is never undefined. 
if isRow === flase, a general name,  like "Anzahl Isoalte", is shown.